package.json

```diff
@@ -9,8 +9,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.0",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.13.2",
+    "@mui/styled-engine-sc": "^5.12.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "styled-components": "^5.3.11"
```

Comment:
This project uses Tailwind for css styling as documented in Vanquish's readme file. 

these styling packages will need to be removed: emotion, emotion/react, emotion/styled, styled-components, @mui-styled-engine-sc

conflict with the established Tailwind based styling convention of Vanquish. the styling of this PR will need to be converted to using Tailwind in order to be merged so as not to have conflicting, duplicate style libraries introduced to the project.

Thanks very much for your PR, I appreciate you taking the time to code up this feature.